src/index.js

```diff
@@ -185,65 +186,82 @@ function toProto(doc) {
   // params
   coll = doc.alps.descriptor.filter(semantic);
   coll.forEach(function(msg) {
-    rtn += `message ${msg.id}Params {\n`;
+    val = makePascalCase(msg.id);
+    rtn += `message ${val}_params {\n`;
```

Comment:
Also capture a camel-case formatted value to be used in the next hunk.
```
val1 = makeCamelCase(msg.id);
```

Reply:
The previous comment has a mistake.
It should be snake-case because it will be used to identify a field.

```
val1 = makeSnakeCase(msg.id);
```